### PR TITLE
Added PEP8Bear and YAMLLintBear to .coafile

### LINKED ISSUE #191: Bear suggestions for repo
It would be good to have bear for checking linting errors in python.
`PEP8Bear` for detecting pep8 incompliant code.
`YAMLLintBear` for checking errors with .yml files

### PR DESCRIPTION
**Pull request by Monal5031**
Fixes #191

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous timespan of 7 days.

### CODE REVIEW

**Comment by aviaryan** [comment]
Comment on d2dacaaa629415be8948e2179b0b0be1b994469b, file stack/stack.py, line 1.

The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,4 +1,5 @@
 class Stack(object):
+
     def __init__(self):
         self._list = []
 
```

**Comment by aviaryan** [comment]
Comment on d2dacaaa629415be8948e2179b0b0be1b994469b, file modular_exponential/modular_exponential.py, line 3.

The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/modular_exponential/modular_exponential.py
+++ b/modular_exponential/modular_exponential.py
@@ -1,6 +1,7 @@
 """
 Modular Exponential
 """
+
 
 def mod_exponent(base, power, mod):
     """
```

**Comment by Monal5031** [comment]
@singhpratyush coala PEP8Bear has issues with some files in repo should I rectify them in my PR itself?

**Comment by aviaryan** [comment]
Yes, please do so.

**Comment by aviaryan** [comment]
Comment on d2dacaaa629415be8948e2179b0b0be1b994469b, file modular_exponential/modular_exponential.py, line 3.

The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/modular_exponential/modular_exponential.py
+++ b/modular_exponential/modular_exponential.py
@@ -1,6 +1,7 @@
 """
 Modular Exponential
 """
+
 
 def mod_exponent(base, power, mod):
     """
```

**Comment by aviaryan** [comment]
Comment on d2dacaaa629415be8948e2179b0b0be1b994469b, file stack/stack.py, line 1.

The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,4 +1,5 @@
 class Stack(object):
+
     def __init__(self):
         self._list = []
 
```

**Comment by Monal5031** [comment]
@aviaryan sir even after making the changes the gitmate bot is raising the issue.

File: modular_exponential/modular_exponential.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Modular Exponential
 """
 
+
```
Comment (singhpratyush) [inline modular_exponential/modular_exponential.py]: Why these changes?

File: stack/stack.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class Stack(object):
+
```
Comment (singhpratyush) [inline stack/stack.py]: And thos one?

**Comment by aviaryan** [comment]
@Monal5031 Notice the first line in gitmate comment.
"Comment on d2dacaa, file stack/stack.py, line 1."
It is still analysing the first commit, this is a bug. For an update report, squash your commits.

File: stack/stack.py (hunk 2 above)
Comment (Monal5031) [inline stack/stack.py]: @singhpratyush sir these were the issues raised by coala, I asked above if I should fix them.

Comment (singhpratyush) [inline stack/stack.py]: Do not anything that is not relevant to the PR.

Comment (Monal5031) [inline stack/stack.py]: Okay will make the changes back but it will continue to occur in other prs also then after this PR is merged as PEP8Bear will point it out in them. Will create a separate issue for that is it okay @singhpratyush sir?

Comment (singhpratyush) [inline stack/stack.py]: Yes, please.

**Comment by aviaryan** [comment]
Comment on abc37724bf5c1e088dd601310e19f7af986026a5, file modular_exponential/modular_exponential.py, line 3.

The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/modular_exponential/modular_exponential.py
+++ b/modular_exponential/modular_exponential.py
@@ -1,6 +1,7 @@
 """
 Modular Exponential
 """
+
 
 def mod_exponent(base, power, mod):
     """
```

**Comment by aviaryan** [comment]
Comment on abc37724bf5c1e088dd601310e19f7af986026a5, file stack/stack.py, line 1.

The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/stack/stack.py
+++ b/stack/stack.py
@@ -1,4 +1,5 @@
 class Stack(object):
+
     def __init__(self):
         self._list = []
 
```
